Maquetado/scss/titles.scss

```diff
@@ -0,0 +1,23 @@
+$gray-dark: #333;
```

Comment:
move the variables to a separate file (`variables.scss`) and import them in the application after the equalizer.